indico_toolkit/association/positioning.py

```diff
@@ -42,6 +42,33 @@ def positioned_above(
             is_above = False
         return is_above
 
+    def positioned_above_overlap(
+            self, above_pos: dict, below_pos: dict, min_overlap_percent: float = None
+        ) -> bool:
+        """
+        Check if the location of one box is above another and if the lower box's overlap is at least the given percentage.
+        Args:
+            above_pos (dict): the position expected to be above
+            below_pos (dict): the position expected to be below
+            min_overlap_percent (float, optional): the minimum amount of overlap needed. Defaults to None.
+
+        Returns:
+            bool: True is above_pos is above below_pos and below_pos' amount of overlap is at least min_overlap_percent
+        """
+        is_above = False
+        is_min_overlap = True
+        if below_pos["page_num"] < above_pos["page_num"]:
+            raise ToolkitInputError(
+                "Predictions are not on the same page!"
+            )
+        if self.xaxis_overlap(above_pos, below_pos):
```

Comment:
I think this just proves that one of the predictions is above the other one, but not necessarily that above_pos is above below_pos, think you need to add:
`self.yaxis_above(above_pos, below_pos):`